packages/api/src/sequelize/index.ts

```diff
@@ -38,18 +40,42 @@ export const getUmzugWithMeta = async (
     migrations: migrations.length,
     executed: executed.length,
     pending: pending.length,
+    lastExecuted: executed[executed.length - 1]?.name,
   };
 };
 
 // export the type helper exposed by umzug, which will have the `context` argument typed correctly
 export type Migration = (params: MigrationParams<QueryInterface>) => Promise<unknown>;
 
 const updateDB = async (sequelize: Sequelize): Promise<MigrationMeta[]> => {
-  const { umzug, migrations, executed, pending } = await getUmzugWithMeta(sequelize);
+  const prefix = `[--- SEQUELIZE ---] `;
```

Comment:
hypernit: if we make prefixes used in template strings not include formatting spaces, it makes the template string more representative of what it actually is.

so, if we have `const prefix = "[--- SEQUELIZE ---]";`

then instead of: `{prefix}Migrations:`
we would have: `{prefix} Migrations:`

Really minor, but it's easier to read this way, and then we're not wondering "did he forget a space here"

Reply:
I intentionally left the space on the template. The reason is that if one makes the prefix and empty string the places that use it won't have a weird empty space before them - the empty space is part of the prefix and not of the actual content.

Reply:
I see - still think the suggested approach is "better", but it's subjective 😄